src/components/button/button.scss

```diff
@@ -31,6 +31,28 @@
   @include md-fab($md-mini-fab-size, $md-mini-fab-padding)
 }
 
+[md-icon-button] {
+  @extend %md-button-base;
+
+  min-width: 0;
+  padding: 0;
+
+  width: 40px;
+  height: 40px;
+
+  line-height: 24px;
+  border-radius: 50%;
```

Comment:
I see that the material1 icon-button is the same, but I'm curious: why does it use `border-radius: 50%`?